src/command_modules/azure-cli-acr/README.rst

```diff
@@ -86,39 +86,39 @@ Update a container registry
         az acr update: Update a container registry.
 
     Arguments
-        --name -n [Required]: Name of container registry.
-        --disable-admin     : Disable admin user.
-        --enable-admin      : Enable admin user.
-        --resource-group -g : Name of resource group.
-        --tags              : Space separated tags in 'key[=value]' format. Use "" to clear existing
-                            tags.
-        --tenant-id -t      : Tenant id for service principal login. Warning: Changing tenant id will
-                            invalidate assigned access of existing service principals.
+        --name -n      [Required]: Name of container registry.
+        --disable-admin          : Disable admin user.
+        --enable-admin           : Enable admin user.
```

Comment:
Generally in the CLI we don't use --enable and --disable flags as it leads to flag clutter and depending on the other arguments, could be separated in the argument list. 

Instead, for creates we would have a single flag which triggers the non-default case (which would be, I assume, enable-admin). Then for the update, we would have an option --admin which would have the choice list `['enable', 'diasable']`